cpp/src/lift.cpp

```diff
@@ -0,0 +1,46 @@
+//
+// Created by Anders Arnholm on 2019-12-14.
+//
+
+#include <sstream>
+#include <iomanip>
+#include "lift.h"
```

Comment:
To get this to compile on Windows with mingw, I found I needed to add the following, to fix errors about `std::copy_if` not being in `std`:
`#include <algorithm>`

Reply:
That should have given me a warning...